common/org.testeditor.dsl.common.testing/src/org/testeditor/dsl/common/testing/ResourceSetHelper.xtend

```diff
@@ -0,0 +1,53 @@
+package org.testeditor.dsl.common.testing
+
+import com.google.inject.Provider
+import javax.inject.Inject
+import org.eclipse.emf.common.util.URI
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.xtext.resource.XtextResourceSet
+import org.testeditor.aml.AmlModel
+import org.testeditor.tcl.TclModel
+import org.testeditor.tsl.TslModel
+import org.eclipse.xtend.lib.annotations.Accessors
+
+@Accessors
+class ResourceSetHelper {
```

Comment:
Sorry maybe I should have been more clear here. I meant a `ParseHelper` that not only incorporates the resource set stuff but also the parse operations.

A lot of tests now inherit from `AbstractParserTest` but they are in fact no parser tests but something else. They simply want to helper methods. Providing a `ParserHelper` with the resource set stuff and the parsing stuff in it would help a lot here.